### PR TITLE
Basic settings

### PR DESCRIPTION
**Pull request by Oleksandr-Adamchuk**
Two tasks have been finished.
Please, make a review.

Task 1 -  Organize the initial GitHub repo
Task 2 - Assemble the list of gems

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
+RSpec.configure do |config|
+
+  config.before(:all) do
+  FactoryBot.reload
```
Comment (romaslmd) [inline spec/spec_helper.rb]: looks like this should be inside rails_helper
as we use factory_bot_rails, so we gonna test all factories per rails env

**Comment by romaslmd** [comment]
looks like shoulda-matchers config is missing

File: config/database.yml.example [hunk 2]
```diff
@@ -1,87 +0,0 @@
-
```
Comment (f1nwe) [inline config/database.yml.example]: you need this file

File: config/secrets.yml.example [hunk 3]
```diff
@@ -1,32 +0,0 @@
-# Be sure to restart your server when you modify this file.
```
Comment (f1nwe) [inline config/secrets.yml.example]: and this too.

File: spec/spec_helper.rb (hunk 1 above)
Comment (Oleksandr-Adamchuk) [reply spec/spec_helper.rb]: Yes, you are right, I've fixed that.

File: config/database.yml.example (hunk 2 above)
Comment (Oleksandr-Adamchuk) [reply config/database.yml.example]: Yes, I know, team member has deleted them with a commit..
Fixed.

File: config/secrets.yml.example (hunk 3 above)
Comment (Oleksandr-Adamchuk) [reply config/secrets.yml.example]: Thanks, fixed  too.

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -0,0 +1,86 @@
+# require database cleaner at the top level
+require 'database_cleaner'
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
```
Comment (romaslmd) [inline spec/rails_helper.rb]: why is this for?

comment saying "Prevent database truncation" but it depends on configuration

**Pull request merged**